attalos/imgtxt_algorithms/demo_app/attalos_demo_app.py

```diff
@@ -0,0 +1,117 @@
+from flask import Flask,request, render_template, send_from_directory
+import numpy as np
+import h5py
+import gzip
+from attalos.util.wordvectors.glove import GloveWrapper
+from os.path import basename
+app = Flask(__name__)
+algo1={'name': 'negsamp fixed 2048,1024,200',
```

Comment:
I think it would be useful to explain how to generate these files (hdf5, id_lookup). Maybe in the README?